server/watchlist.ini

```diff
@@ -1,21 +1,40 @@
-; Example watchlist
```

Comment:
Don't commit changes to the default watchlist.
You can do a "git restore watchlist.ini" to reset your changes before committing.
Or "git update-index --assume-unchanged watchlist.ini" to tell git to stop checking for changes in that file.
Then you can edit the watchlist.ini and have git ignore it when you do "git add -A"

Reply:
Try the following to get this file back to its original state:
`git restore server/watchlist.ini -s main`   
Then git add and commit the file